config/wallet-config.json

```diff
@@ -3,82 +3,93 @@
     "global": []
   },
   "activeFiatProviders": {
+    "coinbase": {
+      "enabled": true,
+      "hasAvailabilityInsideUnitedStates": true,
+      "hasAvailabilityOutsideUnitedStates": false,
+      "hasFastCheckoutProcess": true,
+      "hasTradingFees": true,
+      "name": "Coinbase"
+    },
     "moonPay": {
       "enabled": true,
+      "hasAvailabilityInsideUnitedStates": true,
+      "hasAvailabilityOutsideUnitedStates": true,
       "hasFastCheckoutProcess": true,
-      "name": "MoonPay",
       "hasTradingFees": true,
-      "hasUnitedStatesAvailability": true
+      "name": "MoonPay"
     },
     "transak": {
       "enabled": true,
+      "hasAvailabilityInsideUnitedStates": true,
+      "hasAvailabilityOutsideUnitedStates": true,
```

Comment:
_centre of the world 🌎_ 

This is fine for now if we wanna keep simple, though I wonder if we should do this more like: 

```json5
{
  "availableRegions: ["us", "uk"] // ISO 3166-1
}
```

Reply:
_center_

<img width="397" alt="image" src="https://user-images.githubusercontent.com/28991/184617532-dd5028bd-8263-40d0-b431-88c360519268.png">

Reply:
@kyranjamie what are you asking me to change here? I can't tell from your comment.

Reply:
The API I would be much cleaner if, instead of:

```
      "hasAvailabilityInsideUnitedStates": true,
      "hasAvailabilityOutsideUnitedStates": true,
```

each provider used

```js
{
  "transak": { availableRegions: 'inside-usa-only' }
}
```